core/src/scene/sceneLoader.cpp

```diff
@@ -71,16 +73,11 @@ bool SceneLoader::loadScene(std::shared_ptr<Scene> _scene) {
 void SceneLoader::applyUpdates(Scene& scene, const std::vector<SceneUpdate>& updates) {
     auto& root = scene.config();
     for (const auto& update : updates) {
-        Node value;
         try {
-            value = YAML::Load(update.value);
-        } catch (YAML::ParserException e) {
+            YamlPath(update.path).create(root) = YAML::Load(update.value);
```

Comment:
Maybe good to check if the `update.path` exists in the current `scene.config` before trying the update. Maybe this also fixes the tests.